.github/workflows/linux-nightly.yml

```diff
@@ -54,3 +55,47 @@ jobs:
           for pkg in fixed_dist/torcharrow*.whl; do
               ~/.local/bin/aws s3 cp "$pkg" "$S3_PATH" --acl public-read
           done
+
+  macos-container:
+    runs-on: macos-11
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version:
+          - 3.7
+          - 3.8
+          - 3.9
+    steps:
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Build the wheel
+        run: |
+          ./csrc/velox/velox/scripts/setup-macos.sh
+          pip install wheel
+          ./packaging/build_wheel.sh
+          pip install delocate
+          delocate-wheel dist/*.whl -w fixed_dist
```

Comment:
Technically, we should install `whl` and run tests before uploading to nightly.

Reply:
@ejguan : Right. But the GA Mac machine is old cpu and doesn't have AVX2, it's guaranteed to have "illegal instructions". 

Another reason why make Velox to support non-AVX2 is so critical ^_^. cc @Yuhta, @pedroerp , @damianr99